basis/elevate/macosx/macosx.factor

```diff
@@ -0,0 +1,28 @@
+USING: accessors arrays cocoa.apple-script elevate
+elevate.unix.private formatting io.launcher kernel locals
+sequences system ;
+IN: elevate.macosx
+
+<PRIVATE
+: apple-script-elevated ( command -- )
+    first quote-apple-script
+    "do shell script %s with administrator privileges without altering line endings"
```

Comment:
The ``%s`` here needs to be rethought. We have a command as ``{ "ls" "-al" }`` and that doesn't work with the format string here.

Reply:
https://github.com/barneygale/elevate/blob/master/elevate/posix.py#L37

Reply:
the format argument in the Python is `quote_applescript(quote_shell(args))`, which is supposed to be replaced by `quote-apple-script` but I couldn't find our `quote-shell` word even though i know we have one